src/main/java/de/tum/in/www1/artemis/domain/statistics/StatisticsEntry.java

```diff
@@ -0,0 +1,61 @@
+package de.tum.in.www1.artemis.domain.statistics;
+
+import java.time.temporal.TemporalAccessor;
+
+/**
+ * A data entry used by the statistics pages to calculate the graph data
+ * day is the the timeslot in which the data is created
+ * amount is the number of entries in the same timeslot
+ * username is for specific purposes (Logged-in users, active user, active tutors) where duplicated user entries need to
+ * be filtered out in java and therefore not the amount is needed, but the name so the amount can be calculated afterwards
+ */
+public class StatisticsEntry {
+
+    private TemporalAccessor day;
+
+    private long amount;
+
+    private String username;
+
+    public StatisticsEntry(TemporalAccessor day, long amount) {
+        this.day = day;
+        this.amount = amount;
+        this.username = "";
+    }
+
+    public StatisticsEntry(TemporalAccessor day, String username) {
+        this.day = day;
+        this.amount = 1L;
+        this.username = username;
+    }
+
+    public StatisticsEntry() {
+        this.day = null;
```

Comment:
I guys the null assignment can be removed in case the empty constructor is called as it will implicitly be initialized with null, or am I missing something here?

Reply:
No, thats probably true. I wanted to explicitly set it to make the constructors consistent but its indeed not necessary